tutorials/cifar10-distributed.ipynb

```diff
@@ -0,0 +1,1553 @@
+        "    val_evaluator = create_evaluator(model, metrics, config)\n",
+        "\n",
+        "    def run_validation(engine):\n",
+        "        epoch = trainer.state.epoch\n",
+        "        state = train_evaluator.run(train_loader)\n",
+        "        log_metrics(logger, epoch, state.times[\"COMPLETED\"], \"train\", state.metrics)\n",
+        "        state = val_evaluator.run(val_loader)\n",
+        "        log_metrics(logger, epoch, state.times[\"COMPLETED\"], \"val\", state.metrics)\n",
+        "\n",
+        "    trainer.add_event_handler(\n",
+        "        Events.EPOCH_COMPLETED(every=config[\"validate_every\"]) | Events.COMPLETED,\n",
+        "        run_validation,\n",
+        "    )\n",
+        "\n",
+        "    if rank == 0:\n",
+        "        evaluators = {\"train\": train_evaluator, \"val\": val_evaluator}\n",
+        "        tb_logger = common.setup_tb_logging(\n",
+        "            config[\"output_path\"], trainer, optimizer, evaluators=evaluators\n",
+        "        )\n",
+        "\n",
+        "    best_model_handler = Checkpoint(\n",
+        "        {\"model\": model},\n",
+        "        get_save_handler(config),\n",
+        "        filename_prefix=\"best\",\n",
+        "        n_saved=2,\n",
+        "        global_step_transform=global_step_from_engine(trainer),\n",
+        "        score_name=\"val_accuracy\",\n",
+        "        score_function=Checkpoint.get_default_score_fn(\"Accuracy\"),\n",
+        "    )\n",
+        "    val_evaluator.add_event_handler(\n",
+        "        Events.COMPLETED,\n",
+        "        best_model_handler,\n",
+        "    )\n",
+        "\n",
+        "    try:\n",
+        "        trainer.run(train_loader, max_epochs=config[\"num_epochs\"])\n",
+        "    except Exception as e:\n",
+        "        logger.exception(\"\")\n",
+        "        raise e\n",
+        "\n",
+        "    if rank == 0:\n",
+        "        tb_logger.close()"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "WioiRM5U9ipQ"
+      },
+      "source": [
+        "## Running Distributed Code\n",
+        "\n",
+        "We can easily run the above code with the context manager [Parallel](https://pytorch.org/ignite/generated/ignite.distributed.launcher.Parallel.html#ignite.distributed.launcher.Parallel):\n",
+        "\n",
+        "```python\n",
+        "with idist.Parallel(backend=backend, nproc_per_node=nproc_per_node) as parallel:\n",
+        "    parallel.run(training, config)\n",
+        "```\n",
+        "`Parallel` enables us to run the same code across all supported distributed backends and non-distributed configurations in a seamless manner. It accepts a `backend` (To synchronize all CPUs/GPUs. Find out all supported backends via `ignite.distributed.utils.available_backends()`) and either:\n",
```

Comment:
@Priyansi what do you mean by "To synchronize all CPUs/GPUs." ?

Reply:
I thought I might write the function of `backend` or why we need it. If it's not clear, I can remove that.

Reply:
IMO, both sentences in the brackets are unclear to me
> To synchronize all CPUs/GPUs. Find out all supported backends via ignite.distributed.utils.available_backends()

Either rephrase or remove

Reply:
How about this? -
> `Parallel` enables us to run the same code across all supported distributed backends and non-distributed configurations in a seamless manner. The backend synchronizes all CPUs or GPUs. Read more about which backend to choose [here](https://pytorch.org/docs/stable/distributed.html#backends). `Parallel` accepts a `backend` and either: